### PR TITLE
change alizainy function

### PR DESCRIPTION
**Pull request by Ali873-debug**
change alizainy function

### CODE REVIEW

**Review by AmeerAmjed — CHANGES_REQUESTED** [review]
plase change that yet

File: src/main/main.kt [hunk 1]
```diff
@@ -4,7 +4,7 @@ fun tarek(){
 
 
 
-fun Alizainy() = print("hi ali zainy")
+fun Alizainy() = print("heyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy")
```
[the remark this answers is not in the document]
Comment (Mohammedbih) [reply src/main/main.kt:7]: use for loop :"

**Review by AmeerAmjed — APPROVED** [review]

**Review by Mohammedbih — APPROVED** [review]
عشان واسطة بس

File: src/main/main.kt [hunk 2]
```diff
@@ -4,7 +4,7 @@ fun tarek(){
 
 
 
-fun Alizainy() = print("hi ali zainy")
+fun Alizainy() = print("heyyyyyyyyyyyyyy")
```
Comment (TarekIdrees) [inline src/main/main.kt:7]: Helllllo

**Review by TarekIdrees — APPROVED** [review]

**Pull request merged**